exporter/stats/stackdriver/stackdriver.go

```diff
@@ -101,9 +102,41 @@ type Options struct {
 	Resource *monitoredrespb.MonitoredResource
 }
 
+// Enforces the singleton on NewExporter per projectID per process
+// lest there will be races with Stackdriver.
+var seenProjectIDs struct {
+	m map[string]bool
+	sync.Mutex
+}
+
+func init() {
+	seenProjectIDs.m = make(map[string]bool)
```

Comment:
I'd prefer us to remove seenProjectIDs type. Then initialization will be at the same line:

var (
seenProjectMu sync.Mutex
seenProjects = make(map[string]struct{})
)

Reply:
Great point, thank you.